### ISSUE TITLE
Including license in repository

### ISSUE THREAD

**Issue by kequach** [amices/mice#470]
I checked for the license of this package and found it in the description file and on the website. However, it would be good to add a LICENSE file to make it clear which license is used. The benefit of adding such a file is that it shows up on the "about" section on the right side like this: 
![image](https://user-images.githubusercontent.com/18238845/154534866-6357a73c-f70c-4c9c-adcd-b6ed496663fa.png)

I noticed that you use two licenses which has no clear schema. However, I found this link that showcases some ideas on how to add dual licenses to a repository: https://softwareengineering.stackexchange.com/questions/304874/declaring-multiple-licences-in-a-github-project
If you would like I can create a pull request for this if you tell me what your preference would be.

**Comment by stefvanbuuren** [comment]
Not quite sure what you mean. The licence for the `mice` package is defined in `DESCRIPTION`. 

```
License: GPL-2 | GPL-3
```

Wouldn't that be enough?

**Comment by kequach** [comment]
Yes, it's enough. But it is a bit hidden from this repository. Adding the LICENSE file makes this information easier to find on Github. Not necessary, just a small improvement.

**Comment by stefvanbuuren** [comment]
OK, thanks. 

I remember I had a GPL LICENCE text once, but the CRAN said to remove it (presumably because it is considered redundant). I guess I could include it and leave it out of the package build so that it shows in GitHub.

**Comment by kequach** [comment]
Ah interesting, I didn't know about that. I just know that other R packages include a (redundant) license, for example [Tidyverse](https://github.com/tidyverse/tidyverse). It's also included in the package build which I assumed to be the standard.
